src/main/resources/com/google/api/codegen/java/http_unit_test.snip

```diff
@@ -0,0 +1,137 @@
+@extends "java/common.snip"
```

Comment:
It seems like either this file should be named http_test.snip or the grpc one should be named grpc_unit_test.snip.